app/controllers/idv/gpo_controller.rb

```diff
@@ -250,10 +250,10 @@ def delete_async
       idv_session.idv_gpo_document_capture_session_uuid = nil
     end
 
-    def timed_out
+    def missing
       flash[:info] = I18n.t('idv.failure.timeout')
```

Comment:
should we get and translate a new string so that we don't confuse timeout with missing when we get people writing in?

Reply:
I'm leaning towards re-adding the messages/templates that were removed in https://github.com/18F/identity-idp/pull/5642, which should help distinguish the two cases from one another